packages/valory/skills/transaction_settlement_abci/behaviours.py

```diff
@@ -198,7 +198,7 @@ def _get_tx_data(
 
         # Send transaction
         tx_digest, rpc_status = yield from self.send_raw_transaction(
-            message.raw_transaction
+            message.raw_transaction, use_flashbots
         )
```

Comment:
We only support using/not-using flashbots at the moment as it is not straightforward to also serialize the list of an arbitrary number of block numbers in [hash_payload_to_hex](https://github.com/valory-xyz/open-autonomy/blob/7f3d8d469c4609f63187de244688c047bbd2a5eb/packages/valory/skills/transaction_settlement_abci/payload_tools.py#L91) and [skill_input_hex_to_payload](https://github.com/valory-xyz/open-autonomy/blob/7f3d8d469c4609f63187de244688c047bbd2a5eb/packages/valory/skills/transaction_settlement_abci/payload_tools.py#L154) of the payload tools because they have the `data`, the size of which is not predefined.

Therefore, the default number specified to the ledger (25) will always be used for now as the value of `target_block_numbers`.

However, this can very easily be addressed with [[Refactor] Replace payload_tools with dataclass implementation](https://trello.com/c/MZuy8quS).